src/NuGet.Services.AzureSearch/Db2AzureSearch/Db2AzureSearchCommand.cs

```diff
@@ -195,6 +201,12 @@ private async Task WriteVerifiedPackagesDataAsync(HashSet<string> verifiedPackag
             _logger.LogInformation("Done uploading the initial verified packages data file.");
         }
 
+        private async Task WritePopularityTransfersDataAsync(SortedDictionary<string, SortedSet<string>> popularityTransfers)
```

Comment:
suggest moving this to a common place and call this from `auxiliary2azuresearch` TODO above... the implementation can be TODO, one less thing to worry about.

Reply:
I'm not sure I understand this comment. This code path is separate from `auxiliary2azuresearch`. 

I removed this method since having an empty method is weird though